scripts/setup_kafka.py

```diff
@@ -0,0 +1,53 @@
+"""
+Kafka bootstrap script to create the event_logs topic
+"""
+import os
+import asyncio
+import sys
+
+from aiokafka.admin import AIOKafkaAdminClient, NewTopic
+from aiokafka.errors import TopicAlreadyExistsError, KafkaConnectionError
+
+KAFKA_BOOTSTRAP_SERVERS = os.environ.get("KAFKA_HOST_ADDR", "localhost:9092").split(',')
+KAFKA_TOPIC = "event_logs"
+
+async def main() -> None:
+    """ Setup event_logs if it does not exist """
+    print(f"Connecting to Kafka brokers: {KAFKA_BOOTSTRAP_SERVERS}")
+
+    admin = AIOKafkaAdminClient(bootstrap_servers=KAFKA_BOOTSTRAP_SERVERS)
+
+    try:
+        await admin.start()
+        print("Connected to Kafka successfully")
```

Comment:
Could you also augment and include another call to ensure that indeed it connected the cluster and nodes are operational? Something similar that we did for Mongo https://github.com/kytos-ng/kytos-end-to-end-tests/blob/master/scripts/gitlab_rs_init.py#L67. I want to make sure that indeed the nodes formed the cluster correctly (just so any future test case where it might test node failures we won't have surprises)